utils/calendrical_calculations/src/julian.rs

```diff
@@ -1,99 +1,99 @@
 // This file is part of ICU4X.
 //
-// The contents of this file implement algorithms from Calendrical Calculations
-// by Reingold & Dershowitz, Cambridge University Press, 4th edition (2018),
-// which have been released as Lisp code at <https://github.com/EdReingold/calendar-code2/>
-// under the Apache-2.0 license. Accordingly, this file is released under
-// the Apache License, Version 2.0 which can be found at the calendrical_calculations
-// package root or at http://www.apache.org/licenses/LICENSE-2.0.
-
-use crate::helpers::{i64_to_i32, I32CastError};
+// The contents of this file implement algorithms from the article:
+// "Euclidean Affine Functions and Applications to Calendar Algorithms"
+// by Cassio Neri & Lorenz Schneider (Feb. 2021), DOI: 10.48550/arXiv.2102.06959
```

Comment:
Please refer to the peer reviewed article (it's much nicer to read)
```suggestion
// "Euclidean affine functions and their application to calendar algorithms"
// by Cassio Neri & Lorenz Schneider (Dec. 2022), DOI: 10.1002/spe.3172
// 
```